prow/plugins/config.go

```diff
@@ -817,9 +853,15 @@ func (c *Configuration) EnabledReposForPlugin(plugin string) (orgs, repos []stri
 				repos = append(repos, repo)
 			} else {
 				orgs = append(orgs, repo)
+				orgExceptions[repo] = sets.NewString(plugins.ExcludedRepos...)
```

Comment:
I am not sure `orgExceptions` is the right way to go... do you have an idea @alvaroaleman ?

Reply:
My feeling is that this is very explicit, like `orgA/repoB`, so a slice of repos should do?

Reply:
I am just trying to integrate with the existing mechanisms